android/app/src/googleplay/java/org/fossasia/openevent/utils/ClusterRenderer.java

```diff
@@ -0,0 +1,38 @@
+package org.fossasia.openevent.utils;
+
+import android.content.Context;
+
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.Marker;
+import com.google.android.gms.maps.model.MarkerOptions;
+import com.google.maps.android.clustering.ClusterManager;
+import com.google.maps.android.clustering.view.DefaultClusterRenderer;
+
+import org.fossasia.openevent.R;
+
+/**
+ * Created by rohanagarwal94 on 4/7/17.
+ */
+public class ClusterRenderer extends DefaultClusterRenderer<MicrolocationClusterWrapper> {
```

Comment:
But this utils package is created in the google play package inside the source, just like the map fragment in the fragments package inside the google play package while the other fragments in the main package inside the fragments package. Please suggest the name of the package if it doesn't fit right.